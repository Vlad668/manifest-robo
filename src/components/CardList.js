import React from 'react'
import Card from './Card'


const CardList = ({robotsPassed}) => {
    const RobotListComponent = robotsPassed.map((user, i)=>{
        return (
            <Card key={robotsPassed[i].id} id={robotsPassed[i].id} name={robotsPassed[i].name} email={robotsPassed[i].email}/>
        )
    })
    return (
    <section>
    {RobotListComponent}
    </section>
    )
}

export default CardList