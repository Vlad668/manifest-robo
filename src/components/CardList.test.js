import CardList from './CardList'
import { create } from "react-test-renderer"

const mockData = [
    {
        id:1,
        name:'Test Name',
        email:'test@test.com'
    }
]

const app = create(<CardList robotsPassed={mockData} />);


it('snapwatch test', () => {
    expect(app.toJSON()).toMatchSnapshot()
})