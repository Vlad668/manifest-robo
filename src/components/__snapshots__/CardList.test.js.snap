// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapwatch test 1`] = `
<section>
  <section
    className="tc bg-light-green dib br3 pa3 ma2 grow bw2 shadow-5"
    id="test"
  >
    <img
      alt="profile"
      src="https://robohash.org/1?200x200"
    />
    <div>
      <h3>
        Test Name
      </h3>
      <span>
        test@test.com
      </span>
    </div>
  </section>
</section>
`;
